ios/LinshareShareExtension/ShareViewController.swift

```diff
@@ -20,6 +20,12 @@ class ShareViewController: SLComposeServiceViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        // Clean all files before sharing new file
+        let shareFolderPath = FileManager.default
+            .containerURL(forSecurityApplicationGroupIdentifier: "group.\(hostAppBundleIdentifier)")!
+        let _ = self.cleanAllFilesBeforeSharing(dstURL: shareFolderPath)
+        
+        
```

Comment:
IMO, we should delete temp file after uploaded

Reply:
okay I ll find the way to do it via flutter side

Reply:
@hoangdat @imGok  please check again guys